.github/workflows/rust.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
I saw that the cross_compile jobs failed due to Ubuntu version, but do you know whether this one can stay Ubuntu latest?

Reply:
I'll see if the issue on 24.04 can be fixed, may just be a package that changed its name

Reply:
Fixed!